examples/Streaming/Kafka/MessagePublisher.cs

```diff
@@ -37,7 +37,6 @@ private static void SendMessageBatch(object state)
             });
             var kafkaProducer = new Producer(kafkaProducerConfig);
 */
-            var topicName = "<kafkaTopicName>";
```

Comment:
This is used in line #47 that is commented out because kafkanet library is not yet available in NuGet. Please revert this change